web/auth/auth.go

```diff
@@ -41,12 +44,37 @@ const (
 	Redirect = "redirect"
 )
 
+// Temporary solution. Will be removed when sessions replaced by JWT.
```

Comment:
okay. could you be more precise how much will be replaced; otherwise it is a bit difficult to know how much of this I should "review" properly.

Reply:
Everything session-related will be removed in PR that replaces sessions with JWT. Then the teacher-suffix part will disappear with the introduction of a GitHub App.